test/MusicStore.E2ETests/StoreSmokeTests.cs

```diff
@@ -34,7 +34,7 @@ public async Task DefaultLocation_Kestrel()
                     EnvironmentName = "SocialTesting",
                     PublishApplicationBeforeDeployment = true,
                     PreservePublishedApplicationForDebugging = Helpers.PreservePublishedApplicationForDebugging,
-                    TargetFramework = Tfm.NetCoreApp20, // There's only a Store on 2.0
+                    TargetFramework = Tfm.NetCoreApp30,
```

Comment:
This scenario is not relevant on 3.0 and is already disabled, do not update it. Remove it instead?

Reply:
Yes, the test is stale in that it refers to the runtime store, but I'd rather keep the test and update it to test the shared runtime since the behavior it asserts here is the same with the shared framework. I'll open an issue to update MusicStore tests to test the shared framework.

Reply:
https://github.com/aspnet/AspNetCore/issues/3775